sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosAsyncContainer.java

```diff
@@ -3,37 +3,18 @@
 package com.azure.cosmos;
 
 import com.azure.core.util.Context;
-import com.azure.cosmos.implementation.Constants;
-import com.azure.cosmos.implementation.CosmosPagedFluxOptions;
-import com.azure.cosmos.implementation.Document;
-import com.azure.cosmos.implementation.HttpConstants;
-import com.azure.cosmos.implementation.InternalObjectNode;
-import com.azure.cosmos.implementation.ItemDeserializer;
-import com.azure.cosmos.implementation.Offer;
-import com.azure.cosmos.implementation.Paths;
-import com.azure.cosmos.implementation.RequestOptions;
-import com.azure.cosmos.implementation.TracerProvider;
-import com.azure.cosmos.implementation.Utils;
+import com.azure.cosmos.batch.TransactionalBatchCore;
+import com.azure.cosmos.batch.implementation.BatchAsyncContainerExecutor;
+import com.azure.cosmos.batch.implementation.ItemBatchOperation;
+import com.azure.cosmos.implementation.*;
```

Comment:
NIT - we don't use wildcard imports anywhere intentionally. Please avoid them here (and in other files) for consistency.

Reply:
Ack. Have fixed it at all the place. Thanks.